projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -73,9 +73,8 @@ class Jetpack_Gutenberg {
 	 * @param array  $version_requirements An array containing the required Gutenberg version and, if known, the WordPress version that was released with this minimum version.
 	 * @param string $slug The slug of the block or plugin that has the gutenberg version requirement.
 	 *
-	 * @since 8.3.0
-	 *
 	 * @return boolean True if the version of gutenberg required by the block or plugin is available.
+	 * @since 8.3.0
```

Comment:
Why all the moving around of stuff like this? All these extraneous changes make this PR hard to review, and a few like this one seem contrary to how we normally do things.

Reply:
This was done automatically by the linter when I committed the files, I didn't change this.

Reply:
Was it the linter, or your IDE? When I run the linting locally on trunk it doesn't make these changes, and if I check out this PR and revert them it doesn't redo them when committing. Even when `git reset --soft` back to the merge-base with trunk and recommitting. 🤷

Reply:
Ha, you are spot on – it was my IDE 🤦